.github/workflows/tso-consistency-test.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.20.1"
+          go-version: '1.21'
```

Comment:
```suggestion
          go-version: '1.21.0'
```

Reply:
I intend to use 1.21, which is the equivalent of 1.21.*, to ensure that we can benefit from the minor version.